consultation_analyser/consultations/dummy_data.py

```diff
@@ -72,14 +72,23 @@ def create_multi_choice_answers(question, choices):
     MultiChoiceAnswer.objects.bulk_create(multi_choice_objects)
 
 
-def create_candidate_theme_recursive(question, candidate_theme_data, parent=None):
+def create_candidate_theme_recursive(
+    question, number_respondents, candidate_theme_data, parent=None
+):
     """Create a CandidateTheme (and SelectedTheme if selected) recursively."""
     name = candidate_theme_data.get("name")
     description = candidate_theme_data.get("description", "")
     key = candidate_theme_data.get("key")
+    approximate_frequency = round(
+        candidate_theme_data.get("approximate_frequency_pct") * number_respondents
```

Comment:
```suggestion
        candidate_theme_data.get("approximate_frequency_pct", 0) * number_respondents
```
otherwise you could get a `ValueError`